WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,59 @@
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Data.EF;
+using WebApiCoreSeed.Data.Models;
+using WebApiCoreSeed.Domain.Services;
+using Xunit;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    public class UserServiceTests
+    {
+        [Fact]
+        public async Task GetByIdAsync_ShouldGetUserById_WhenUserIdExists()

```

Comment:
> GetByIdAsync_**ShouldGetUserById**_WhenUserIdExists

**ShouldReturnUser** would be more appropriate/precise.. 

Same goes for the method below:

> GetByIdAsync_**ShouldGetUserById**_WhenUserIdDoesNotExists

Could be **ShouldReturnNull** , otherwise the title of the UT is confusing.. how the method would **GetUserById** if the user does not exist? 😄